### DISCUSSION TITLE
Command Palette Feedback

### DISCUSSION THREAD

**Discussion by jasoncavett** [community/community#15398] [Other Feature Feedback, Questions, & Ideas]
This is a really nice feature from my limited use. Makes it easy to quickly navigate around without lifting my hands from the keyboard. Two things I have noticed in my early usage:

1. I cannot navigate to any other beta items. So, my Projects Beta projects can't be accessed by this method. I am assuming that would change when Projects comes out of Beta, but given that the beta projects already have some navigation issues, this feels like exacerbates things.
2. When I hit Ctrl + K, the default search is in the current repository which may be a good base case, but if an organization has multiple repos (e.g. libraries that are used as part of a larger project) it would be nice to be able to move between those. Or, a simpler scenario, move between a project, an issue screen, and actions. Right now, I have to move my hand to the mouse to make that happen. However, I definitely acknowledge that I haven't spent a ton of time seeing if I was just not understanding this part well yet.

Overall, I think this is a great idea for a feature and look forward to continued iterations!

**Comment by ChristianTacke** [comment]
About (2): Have you tried the Backspace key? This allows you to go one level up and search there. Haven't used it a lot, but seems to work mostly for me. I admit, that this key is not very "discoverable".
